src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataServiceTest.java

```diff
@@ -499,4 +502,24 @@ public void saveCaseEventInterlocatoryJudgement() {
         verify(coreCaseDataApi, atLeastOnce()).startEventForCitizen(anyString(), anyString(), anyString(), anyString(),
             anyString(), anyString(), eq(INTERLOCATORY_JUDGEMENT.getValue()));
     }
+
+    @Test
+    public void saveCaseReDetermination() {
+        ReDetermination reDetermination = ReDetermination.builder()
```

Comment:
// Given when then is missed. I dont like it though, but If you want to fix it, fix it. I am okay without it as well.

Reply:
need refactoring across. so leaving for the time